templates/kubernetes-cluster.template

```diff
@@ -295,6 +295,52 @@ Resources:
   K8sMasterInstance:
     Type: AWS::EC2::Instance
     DependsOn: ApiLoadBalancer
+    Metadata:
+      AWS::CloudFormation::Init:
+        configSets:
+          master-setup: master-setup
+        master-setup:
+          files:
+            # Configuration file for the cloudwatch agent. The file is a Mustache template, and we're creating it with
+            # the below context (mainly to substitute in the AWS Stack Name for the logging group.)
+            "/tmp/kubernetes-awslogs.conf":
+              source: !Sub "https://${QSS3BucketName}.s3.amazonaws.com/${QSS3KeyPrefix}/scripts/kubernetes-awslogs.conf"
+              context:
+                StackName:
```

Comment:
Yeah I was gonna save that for one big commit to do !Ref/!GetAtt/etc everywhere.  But I'll stop writing new code that does it.